app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -580,22 +580,36 @@ const SendBitcoinDetailsScreen: React.FC<Props> = ({ route }) => {
     )
   }
 
+  const isLnurlAmountEditable = lnurlParams && lnurlParams.min === lnurlParams.max
```

Comment:
I think these changes should be made in `createLnurlPaymentDetails`. Where `canSetAmount` should be false if `lnurlParams.min === lnurlParams.max` and the amount should be prepopulated.